x-pack/filebeat/module/shorewall/log/manifest.yml

```diff
@@ -0,0 +1,13 @@
+module_version: 1.0
+
+var:
+  - name: paths
+    default:
+      - /var/log/shorewall.log
+    os.darwin:
+      - /usr/local/example/test.log*
```

Comment:
If we don't know the paths for darwin / windows lets rather remove it.

Reply:
My fault, removing those now